runtime/js/40_write_file.js

```diff
@@ -68,6 +68,7 @@
 
         // always throw the abort error when aborted
         options.signal.throwIfAborted();
+        core.tryClose(cancelRid);
```

Comment:
Need to coordinate cancel handle clearing solutions with https://github.com/denoland/deno/pull/17736

Reply:
With the changes in that PR, these should now be unnecessary additions.